salt/top.sls

```diff
@@ -1,5 +1,7 @@
 base:
-  '*': []
+  '*':
+    - match: grain
```

Comment:
The match parameter is not needed here since the match is being done with globbing.